### ISSUE TITLE
Calculation of numTrainSamplesPerEpoch & batchSize Value

### ISSUE THREAD

**Issue by contactvpatel** [githubharald/SimpleHTR#47]
* TensorFlow version 1.12
* Python version 3.5
* Operating system - Windows 10

We have numTrainSamplesPerEpoch as 25000 set in DataLoader.py and batchSize as 50 in Model.py. In my case, My input image count varies when I am training my model every time, so what's formula / best way to calculate numTrainSamplesPerEpoch & batchSize based on input image count to get best possible accuracy.

**Comment by githubharald** [comment]
Try different values ("hyperparameter search") and see how the results change. There is no formula to get these parameters.
